components/vision.py

```diff
@@ -128,7 +128,7 @@ def execute(self) -> None:
             self.chassis.estimator.addVisionMeasurement(
                 vision_pose,
                 self.vision_data.timestamp,
-                [pos_std_dev, pos_std_dev, 0.0],
+                (pos_std_dev, pos_std_dev, 0.0),
```

Comment:
The type hint says it's a tuple (the bindings do accept a list, but this makes the type checker happy).